### PR TITLE
Fix for KSP version #2

### PR DESCRIPTION
**Pull request by jenyaza01**

### CODE REVIEW

**Comment by jenyaza01** [comment]
Great, i can fix without new pull requesting )

File: NetKAN/WaterTanks.netkan [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "spec_version" : "v1.4",
+	"name"         : "Ballast Water Tanks",
+	"identifier"   : "WaterTanks",
+	"abstract"     : "Add parts with in-flight mass changing. Designed for submarines, act good as payload weight",
+	"author"       : "Jenyaza01",
+    "$kref"        : "#/ckan/github/jenyaza01/BallastWaterTanks",
+	"$vref"        : "#/ckan/ksp-avc",
```
Comment (politas) [inline NetKAN/WaterTanks.netkan]: There's no .version file in the download, so this is not giving netkan any version info. You'll need to specify it manually or add an AVC .version file to the download.

**Comment by jenyaza01** [comment]
I will add .version file in next release, just 1-2 hours

**Comment by politas** [comment]
Looks good to me. Merging with many thanks!

**Pull request merged**